dallinger/experiment_server/experiment_server.py

```diff
@@ -572,11 +573,16 @@ def create_participant(worker_id, hit_id, assignment_id, mode):
         'participant': participant.__json__()
     }
 
+    exp = Experiment(session)
+
+    # Ping back to the recruiter that one of their participants has joined:
+    recruiter = Recruiter.for_experiment(exp)
+    recruiter.notify_recruited(participant, exp)
```

Comment:
Does this return quickly enough to include directly in a web route, as opposed to needing to use the notification system / Redis queue / worker processes?